src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -72,444 +104,413 @@ const Mentor = ({ dispatch, error, successPage }) => {
-                      onChange={e => handleChange(e, 'select', 'country')}
+                      onChange={e => handleChange(e, 'select', 'state')}
                     >
-                      {countries.map(country => (
-                        <Option key={country} value={country}>
+                      {states.map(state => (
+                        <Option key={state} value={state}>
                           {' '}
-                          {country}{' '}
+                          {state}{' '}
                         </Option>
                       ))}
                     </Select>
                   </Form.Item>
-                </Col>
-              </Row>
-
-              <Row>
-                <Col md={15} xs={24} offset={1}>
-                  <div className="locationUS">
-                    <Form.Item
-                      label="State"
-                      style={{ margin: '.5rem 1rem 1rem 0' }}
-                      name="state"
-                      rules={[
-                        {
-                          required: true,
-                          message: 'State is required!',
-                        },
-                      ]}
-                    >
-                      <Select
-                        showSearch
-                        placeholder="- Select -"
-                        onChange={e => handleChange(e, 'select', 'state')}
-                      >
-                        {states.map(state => (
-                          <Option key={state} value={state}>
-                            {' '}
-                            {state}{' '}
-                          </Option>
-                        ))}
-                      </Select>
-                    </Form.Item>
-                  </div>
-
-                  <Form.Item
-                    label="City"
-                    type="text"
-                    name="city"
-                    rules={[
-                      {
-                        required: true,
-                        message: 'City is required!',
-                      },
-                    ]}
-                    value={formValues.city}
-                    onChange={handleChange}
-                    style={{ margin: '0 1rem .5rem 0' }}
-                  >
-                    <Input placeholder="City" />
-                  </Form.Item>
-                </Col>
-              </Row>
+                </div>
 
-              <hr />
+                <Form.Item
+                  label="City"
+                  type="text"
+                  name="city"
+                  rules={[
+                    {
+                      required: true,
+                      message: 'City is required!',
+                    },
+                  ]}
+                  value={formValues.city}
+                  onChange={handleChange}
+                >
+                  <Input placeholder="City" />
+                </Form.Item>
+              </Col>
+            </Row>
```

Comment:
Similarly in this section, I think city, then state, then country would be a more intuitive order.